src/services/GooglePlacesService.ts

```diff
@@ -43,6 +43,23 @@ class GooglePlacesService {
     const place = await this.getRestaurantDetails(restaurantId);
     return place.photos ?? [];
   }
+
+  /**
+   * Constructs and returns the url for a restaurant photo.
+   * Note: retrieves the photo with the largest max width/height for the best quality.
+   *
+   * @param photo the restaurant photo
+   */
+  getRestaurantPhotoUrl({ photo_reference }: PlacePhoto): string {
+    const baseUrl = 'https://maps.googleapis.com/maps/api/place/photo?';
+    const queryParams = new URLSearchParams({
+      maxheight: '1600',
+      maxwidth: '1600',
+      photoreference: photo_reference,
+      key: process.env.REACT_APP_GOOGLE_PLACES_API_KEY as string,
```

Comment:
I remember reading that we should use a separate google places API key that can only be used for image display purposes, (can't be used for place details or any other stuff). Because otherwise, this API key is accessible by any user, what do you think?

Reply:
Ahh yeah, good point I forgot. Thanks for bringing this up!